webevd/WebEVD/WebEVDServer.cxx

```diff
@@ -502,6 +502,9 @@ SerializeProduct(const TEvt& evt, JSONFormatter& json)
     typename TEvt::template HandleT<std::vector<TProd>> prods; // deduce handle type
     evt.getByLabel(tag, prods);
 
+    // This can happen in case of dropped prroducts
+    if(!prods.isValid()) continue;
```

Comment:
This looks fine, Chris.  You can express this more concisely as:

```diff
   typename TEvt::template HandleT<std::vector<TProd>> prods; // deduce handle type
- evt.getByLabel(tag, prods);
- 
- // This can happen in case of dropped prroducts
- if(!prods.isValid()) continue;
+ if(!evt.getByLabel(tag, prods)) continue;
```
and elsewhere (i.e. `getByLabel` returns `true` or `false` if the filled-in handle is valid or invalid, respectively).